Sources/MapboxNavigation/RouteVoiceController.swift

```diff
@@ -172,15 +172,16 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     }
         
     @objc open func didPassSpokenInstructionPoint(notification: NSNotification) {
-        let routeProgress = notification.userInfo![RouteController.NotificationUserInfoKey.routeProgressKey] as! RouteProgress
+        guard let routeProgress = notification.userInfo?[RouteController.NotificationUserInfoKey.routeProgressKey] as? RouteProgress else { return }
```

Comment:
This would be another place to assert in Debug configuration to avoid silently cascading an internal consistency error.